examples/azure_exocompute/main.go

```diff
@@ -82,25 +85,25 @@ func main() {
 		fmt.Printf("Feature: %v, Regions: %v, Status: %v\n", feature.Name, feature.Regions, feature.Status)
 	}
 
-	// Add exocompute config for the account.
-	exoID, err := azureClient.AddExocomputeConfig(ctx, azure.CloudAccountID(accountID),
-		azure.Managed("eastus2", "/subscriptions/9318aeec-d357-11eb-9b37-5f4e9f79db5d/resourceGroups/terraform-test/providers/Microsoft.Network/virtualNetworks/terraform-test/subnets/default"))
+	// Add an exocompute configuration to the cloud account.
+	configID, err := exoClient.AddAzureConfiguration(ctx, accountID,
+		exocompute.AzureManaged(gqlazure.RegionEastUS2, "/subscriptions/9318aeec-d357-11eb-9b37-5f4e9f79db5d/resourceGroups/terraform-test/providers/Microsoft.Network/virtualNetworks/terraform-test/subnets/default"))
```

Comment:
Same question applies here regarding hard-coded values?

Reply:
They are made up placeholders for the example code. Anybody wanting to run the code has to put in their own valid values.